tools/file_packager.py

```diff
@@ -1094,12 +1094,12 @@ def generate_js(data_target, data_files, metadata):
       }\n'''
 
   ret += '''
-    function runWithFS() {\n'''
+    function runWithFS(Module) {\n'''
   ret += code
   ret += '''
     }
     if (Module['calledRun']) {
-      runWithFS();
+      runWithFS(Module);
     } else {
       if (!Module['preRun']) Module['preRun'] = [];
       Module["preRun"].push(runWithFS); // FS is not initialized yet, wait for it
```

Comment:
What about this case?  Wouldn't we need to pass in the module here too?    Maybe we can make a closure here to capture the internal `Module` ?

Reply:
`callRuntimeCallbacks` already passes the internal `Module` in to `preRun` (etc) callbacks, so we should not need to do anything else here?

Reference: https://github.com/emscripten-core/emscripten/blob/404fde04e762fbe12e90e9ca56463bdbb9730ab1/src/library.js#L2832-L2837

Reply:
Great!